server/app/utils.py

```diff
@@ -319,9 +319,13 @@ def add_to_grading_queues(assign_key, cursor=None, num_updated=0):
     seen = set()
     for queue in queues:
         for subm in queue.submissions:
-            seen.add(subm.get().submitter.id())
+            try:
```

Comment:
Instead of a try/except - it should be possible to see if subm is an instance of a particular class (models.Submission?) - (we do this occasionally in models.py/api.py) 

If that's not possible, then I guess a try/except is fine - it's more a style/preference thing to avoid try/excepts.